R/derive_params_growth_age.R

```diff
@@ -116,10 +116,11 @@
 #' library(lubridate, warn.conflicts = FALSE)
 #' library(rlang, warn.conflicts = FALSE)
 #' library(admiral, warn.conflicts = FALSE)
+#' library(pharmaversesdtm, warn.conflicts = FALSE)
 #'
-#' advs <- dm_peds %>%
+#' advs <- pharmaversesdtm::dm_peds %>%
```

Comment:
should be able to remove `pharmaversesdtm::` - i don't think we usually include these. i guess this was just for testing purposes

Reply:
to demonstrate it works being in pharmaversesdtm if its actually in there, yes